tutorial/20_recipes/012_artifact_tutorial.py

```diff
@@ -139,6 +139,13 @@ def objective(trial: optuna.Trial) -> float:
 
     print(content)
 
+Also, you can easily download the artifact as a file using `download_artifact` function, instead of using the artifact module:
```

Comment:
I have two suggestions.

1. Could you remove the code that uses `artifact_store.open_reader()` since it is not a public api? It was just a workaround for `download_artifact()`.
2. Could you add a link on the `download_artifact` to the api doc like below?

```suggestion
Also, you can easily download the artifact as a file using :func:`~optuna.artifacts.download_artifact` function, instead of using the artifact module:
```

Reply:
Thank you for your review! I have removed `open_reader()` from the tutorial.